dockerfiles/init/manifests/che.env

```diff
@@ -468,11 +468,11 @@ CHE_SINGLE_PORT=false
 
 #
 CHE_KEYCLOAK_OSO_ENDPOINT=NULL
-CHE_KEYCLOAK_GITHUB.ENDPOINT=NULL
-CHE_KEYCLOAK_AUTH-SERVER-URL=http://172.17.0.1:5050/auth
+CHE_KEYCLOAK_GITHUB_ENDPOINT=NULL
+CHE_KEYCLOAK_AUTH__SERVER__URL=http://172.17.0.1:5050/auth
 CHE_KEYCLOAK_REALM=che
-CHE_KEYCLOAK_CLIENT-ID=che-public
+CHE_KEYCLOAK_CLIENT__ID=che-public
```

Comment:
do we need double underscore ?  (client_id vs client.id )

because for private client it's `che.keycloak.private.client_id`
(it's only a question)

Reply:
yes, we need it for OpenShift (`-` or `.` are not valid for Openshift in environment variable names). And the `__` of the environment variable maps to the `_` of the property in Che afaik.